internal/app/app.go

```diff
@@ -135,17 +143,24 @@ func (p *PolicyAutomationApp) ClusterReview() error {
 		evalResult.ClusterName = clusterName
 		evalResults = append(evalResults, evalResult)
 	}
-	err = p.collector.RegisterResult(evalResults)
-	if err != nil {
-		p.out.ErrorPrint("failed to register evaluation results", err)
-		log.Errorf("could not register evaluation results: %s", err)
-		return err
+
+	for _, c := range p.collectors {
+		err = c.RegisterResult(evalResults)
+
+		if err != nil {
+			p.out.ErrorPrint("failed to register evaluation results", err)
+			log.Errorf("could not register evaluation results: %s", err)
+			return err
+		}
 	}
-	err = p.collector.Close()
-	if err != nil {
-		p.out.ErrorPrint("failed to close results registration", err)
-		log.Errorf("could not finalize registering evaluation results: %s", err)
-		return err
+
+	for _, c := range p.collectors {
+		err = c.Close()
```

Comment:
I think it will be nice to add some `log.Info(...)` or similar about flushing given collector for better observability.